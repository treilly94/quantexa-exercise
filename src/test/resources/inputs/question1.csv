transactionId,accountId,transactionDay,category,transactionAmount
1,10,1,a,1
2,20,1,a,1
3,30,1,a,1
4,40,1,a,1
5,50,1,a,1
6,60,2,b,1
7,70,2,w,1
8,80,2,e,1
9,90,2,r,1
10,100,2,e,1
11,110,3,c,1
12,120,3,d,2
13,130,3,f,3
14,140,3,s,4
15,150,3,a,5
