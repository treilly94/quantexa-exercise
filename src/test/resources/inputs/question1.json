{"transactionId":"1","accountId":"10","transactionDay":1,"category":"a","transactionAmount":1.0},
{"transactionId":"2","accountId":"20","transactionDay":1,"category":"a","transactionAmount":1.0},
{"transactionId":"3","accountId":"30","transactionDay":1,"category":"a","transactionAmount":1.0},
{"transactionId":"4","accountId":"40","transactionDay":1,"category":"a","transactionAmount":1.0},
{"transactionId":"5","accountId":"50","transactionDay":1,"category":"a","transactionAmount":1.0},
{"transactionId":"6","accountId":"60","transactionDay":2,"category":"b","transactionAmount":1.0},
{"transactionId":"7","accountId":"70","transactionDay":2,"category":"w","transactionAmount":1.0},
{"transactionId":"8","accountId":"80","transactionDay":2,"category":"e","transactionAmount":1.0},
{"transactionId":"9","accountId":"90","transactionDay":2,"category":"r","transactionAmount":1.0},
{"transactionId":"10","accountId":"100","transactionDay":2,"category":"e","transactionAmount":1.0},
{"transactionId":"11","accountId":"110","transactionDay":3,"category":"c","transactionAmount":1.0},
{"transactionId":"12","accountId":"120","transactionDay":3,"category":"d","transactionAmount":2.0},
{"transactionId":"13","accountId":"130","transactionDay":3,"category":"f","transactionAmount":3.0},
{"transactionId":"14","accountId":"140","transactionDay":3,"category":"s","transactionAmount":4.0},
{"transactionId":"15","accountId":"150","transactionDay":3,"category":"a","transactionAmount":5.0}
