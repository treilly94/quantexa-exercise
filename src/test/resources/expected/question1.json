{"transactionDay":1,"totalValue":5.0},
{"transactionDay":2,"totalValue":5.0},
{"transactionDay":3,"totalValue":15.0}